sdk/maps/azure-maps-search/azure/maps/search/aio/_search_client_async.py

```diff
@@ -0,0 +1,131 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+# pylint: disable=unused-import,ungrouped-imports
+from typing import TYPE_CHECKING
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.exceptions import HttpResponseError
+from azure.core.credentials import AzureKeyCredential
+from ._base_client_async import AsyncMapsSearchClientBase
+from .._generated.models import BatchRequest, SearchAddressBatchResult
+
+if TYPE_CHECKING:
+    from typing import Any, List
+    from azure.core.credentials_async import AsyncTokenCredential
+    from azure.core.polling import LROPoller, AsyncLROPoller
+
+
+# By default, use the latest supported API version
+class MapsSearchClient(AsyncMapsSearchClientBase):
```

Comment:
This class seems to be missing all the methods that are in the sync MapsSearchClient?

Reply:
Added all the methods as sync clients.